README.md

```diff
@@ -8,3 +8,21 @@ Required tokens:
 - WICKRIO_BOT_NAME
 - WHITELISTED_USERS - Comma-separated list of wickr users that will be allowed to use the bot
 - DATABASE_ENCRYPTION_KEY - Choose a 16-character(minimum) string key to derive the crypto key from in order to encrypt and decrypt the user database of this bot. This must be specified, there is no default. NOTE: be careful not to change if reconfiguring the bot or else the user database won't be accessible.
+
```

Comment:
Can you please add more details on what needs to be added to the package.json file to get this to work.  Please add enough details so that someone with no knowledge of this can do it.